repos/system_upgrade/common/actors/reportleftoverpackages/reportleftoverpackages/libraries/reportleftoverpackages.py

```diff
@@ -0,0 +1,50 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import LeftoverPackages, RemovedPackages
+
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def process():
+    removed_packages = next(api.consume(RemovedPackages), None)
+    leftover_packages = next(api.consume(LeftoverPackages), LeftoverPackages())
+    leftover_pkgs_to_remove = ['-'.join([pkg.name, pkg.version, pkg.release]) for pkg in leftover_packages.items]
+
+    if removed_packages and removed_packages.items:
+        title = 'Leftover RHEL packages have been removed'
+        removed = ['-'.join([pkg.name, pkg.version, pkg.release]) for pkg in removed_packages.items]
+        summary = (
+            'Following packages have been removed:{sep}{list}\n'
+            'Dependent packages may have been removed as well, please check that you are not missing '
+            'any packages.'
+            .format(
+                sep=FMT_LIST_SEPARATOR,
+                list=FMT_LIST_SEPARATOR.join(removed)
+            )
+        )
+        reporting.create_report([
+            reporting.Title(title),
+            reporting.Summary(summary),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups([reporting.Groups.SANITY]),
+        ] + [reporting.RelatedResource('package', pkg.name) for pkg in removed_packages.items])
+        return
+
+    if leftover_packages and leftover_packages.items:
+        summary = (
+            'Following RHEL packages have not been upgraded:{sep}{list}'
```

Comment:
```suggestion
            'Following RHEL packages have not been upgraded:{sep}{list}\n'
```

Reply:
added missing new line. orig report looked like this (raw output):
```
Following RHEL packages have not been upgraded:\n    - kernel-modules-4.18.0-527.el8\n    - iptstate-2.2.6-6.el8\n    - kernel-4.18.0-553.el8_10\n    - kernel-core-4.18.0-527.el8\n    - kernel-4.18.0-527.el8\n    - kernel-modules-4.18.0-553.el8_10\n    - libmodman-2.0.1-17.el8\n    - kernel-core-4.18.0-553.el8_10Please remove these packages to keep your system in supported state.
```

Reply:
Thanks!